src/ui/map.js

```diff
@@ -1002,9 +1002,9 @@ class Map extends Camera {
     /** @section {Point conversion} */
 
     /**
-     * Returns a {@link ProjectionSpecification} object that defines the current map projection.
+     * Returns a [projection](https://www.mapbox.com/mapbox-gl-style-spec/#projection) object that defines the current map projection.
```

Comment:
This should work but maybe let's switch to https://docs.mapbox.com/mapbox-gl-js/style-spec/projection/ to avoid the redirect